tests/rhui3_tests/test_client_management.py

```diff
@@ -33,7 +33,6 @@ class TestClient:
     @classmethod
     def setUpClass(cls):
         cls.rhua_os_version = Util.get_rhua_version(connection)
-        print "*** Running %s: *** " % basename(__file__)
```

Comment:
This line wasn't needed, so I removed it. The message is already printed earlier in `setUp()`, so it was printed twice before this change.